packages/imperative/src/events/src/doc/IImperativeRegisteredAction.ts

```diff
@@ -0,0 +1,23 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+/**
+ * Imperative Registered Action
+ * @export
+ * @interface IImperativeRegisteredAction
+ */
+export interface IImperativeRegisteredAction {
```

Comment:
If the intent is to implement a disposable, would it make sense to name this something else like `IDisposableAction`?

I was also discussing with @ATorrise and we were wondering if the `Imperative` prefix is necessary on all the class and interface names related to the event emitter. I don't think we include it in most of our class names except for `ImperativeConfig` 😋

Reply:
I agree with both points.
Addressed in 9305b3e1a34fa4336718fb4ce481efc2d7898952